packages/smooth_app/lib/pages/product/product_image_viewer.dart

```diff
@@ -107,17 +108,34 @@ class _ProductImageViewerState extends State<ProductImageViewer> {
                           label:
                               appLocalizations.edit_photo_unselect_button_label,
                           onPressed: () async {
-                            final NavigatorState navigatorState =
-                                Navigator.of(context);
-                            await BackgroundTaskUnselect.addTask(
-                              _barcode,
-                              imageField: widget.imageField,
-                              widget: this,
+                            final bool? includeLogs = await showDialog<bool>(
+                              context: context,
+                              builder: (BuildContext context) {
+                                return SmoothAlertDialog(
+                                    title:
+                                        appLocalizations.confirm_button_label,
+                                    body: Text(
+                                      appLocalizations.asking_confirmation,
+                                    ),
+                                    close: true,
+                                    positiveAction: SmoothActionButton(
+                                        text: appLocalizations.yes,
+                                        onPressed: () =>
+                                            Navigator.of(context).pop(true)),
+                                    negativeAction: SmoothActionButton(
+                                        text: appLocalizations.no,
+                                        onPressed: () =>
+                                            Navigator.of(context).pop(false)));
+                              },
                             );
-                            _localDatabase.notifyListeners();
-                            navigatorState.pop();
-                          },
-                        ),
+                            if (includeLogs ?? false) {
```

Comment:
```suggestion
                            if (includeLogs == true) {
```
would be easier to read.